src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -28,6 +29,13 @@ import zxcvbn from 'zxcvbn';
 import './SignUp.css';
 
 export default class SignUp extends Component {
+  componentDidMount() {
+    const cookies = new Cookies();
```

Comment:
create the cookies outside the class defination. That would look more consistent to the rest of the code base.